index.html

```diff
@@ -0,0 +1,748 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>New-York Times</title>
+    <script
+      src="https://kit.fontawesome.com/5b1eeceadd.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Libre+Franklin&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Domine&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+
+  <!--This is a header section and it includes nav -->
+
+    <header>
+      <nav class="nav flex">
+        <div class="first flex flex-prop">
+          <i class="fas fa-bars mr"></i>
+          <i class="fas fa-search mr"></i>
+          <a href="">Space & Cosmos</a>
+        </div>
+
+        <div class="logo flex-prop">
+          <img src="images/nyt.png" alt="newyorktimes" class="nyt" />
```

Comment:
- [ ] The NY Times logo is a link (link to page), please add `<a>` tag for the NYT logo.